server/market/bookrouter.go

```diff
@@ -118,11 +127,12 @@ func (book *msgBook) remove(lo *order.LimitOrder) {
 	delete(book.orders, lo.ID())
 }
 
-// addBulkOrders adds the lists of orders to the order book. Use this for the
-// initial sync of the orderbook.
-func (book *msgBook) addBulkOrders(orderSets ...[]*order.LimitOrder) {
+// addBulkOrders adds the lists of orders to the order book, and records the
+// currently active epoch. Use this for the initial sync of the orderbook.
+func (book *msgBook) addBulkOrders(epoch int64, orderSets ...[]*order.LimitOrder) {
```

Comment:
This may also prove to be unnecessary for the same reason, with `book.epoch` being set on `newEpochAction` signals.